### ISSUE TITLE
kowab.it list

### ISSUE THREAD

**Issue by pwab** [fboes/fritzbox-blacklist#13]
I stumbled upon another list that might be compatible with fritzboxes: https://list.kwbt.de/fritzboxliste.txt
It's made by [kowab.it](https://kowabit.de/blcklst/) and the last update was on [25.06.2021](https://kowabit.de/update-grosse-blockliste-25-06-2021/) so it seems to be maintained.

Not sure if it has the same purpose as this list here because there are also a lot of entries for 'normal' social media and cdn domains. But maybe there are some relevant domains that could be added here too.

**Comment by 0n1cOn3** [comment]
Easiest part could be to compare the list and links and add the missing ones :)

**Comment by pwab** [comment]
Yeah @0n1cOn3 it was so easy that I did it in a few seconds with notepad++ and its sorting and duplicate lines features. 😅 
Here is the result: https://gist.github.com/pwab/e1ca8de561f7a8d76a7e95b878911573

I removed the social media, cdn, shortlink and payment services from the kowab.it-list so it only contains the _normal_ entries which shouldn't have an effect on pages some people use everyday.
